docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
+### How do I build `blockfrost-platform` from source?
 
-## Can I transfer my Icebreaker NFT to another wallet address?
+Refer to the [installation instructions](/installation/source) for detailed steps on building from source.
+This is often recommended over using pre-built installers if you encounter issues with the provided binaries.
 
-Yes, you can transfer your Icebreaker NFT to another wallet address by following these steps:
+### Do I need a static public IP address for my Icebreaker node?
+
+Yes, a static public IP is strongly recommended for reliably running an Icebreaker node.
+
+The Blockfrost platform currently identifies and monitors nodes based on their consistent public IP. Without a static IP, you risk connection issues and flapping.
+
+A future update will allow dynamic IPs by having the node proactively report its address. See [this GitHub issue](https://github.com/blockfrost/blockfrost-platform/issues/178) for details on that roadmap.
+
+In the interim, using a static IP is best. If not possible, a dynamic DNS service or VPN may help provide a stable hostname to IP mapping.
+
+### What should I do if my public IP address changes frequently?
+
+Frequent public IP address changes can disrupt the connection between your Icebreaker and the Blockfrost servers.
+A future update will address this by allowing connections from behind NATs (see [this GitHub Issue](https://github.com/blockfrost/blockfrost-platform/issues/178)).
+In the meantime, consider using a static IP address or a dynamic DNS service.
+
+### How do I run blockfrost-platform with only IPv4?
+
+While a dedicated `-4` CLI switch isn't currently available, you can work around IPv6 issues by adding an entry to your `/etc/hosts` file that maps the Blockfrost server address to its IPv4 address.
+
+### Should I run my Icebreaker node on bare metal or in a VM?
+
+Either bare metal or VM-based Icebreaker nodes will work fine.
+The choice mainly depends on your available infrastructure and operational preferences.
+
+Benefits of bare metal:
+- Avoids virtualization overhead for maximum performance
+- Provides direct access to hardware for optimization
+- Keeps node infrastructure separate from other workloads
+
+Benefits of VMs:
+- Easier to deploy and manage using common VM platforms
+- Allows sharing server resources with other workloads
+- Provides snapshot and migration capabilities for easier updates
+
+Regardless of your approach, refer to the [Getting started](get-started) documentation to check the prerequisites.
+Over-provisioning CPU, RAM and disk will give you more headroom.
+
+### What is the Blockfrost Platform `server-address` configuration parameter used for?
+
+The `server-address` configuration parameter defines the IP address and port that your Icebreaker node's API server listens on.
+
+By default this uses `0.0.0.0:3000` which listens on all IPv4 interfaces on port 3000.
```

Comment:
This is confusing and incorrect. The port value is provided separately. We don't use this format.
In the config wizard, address and port is provided separately.
In the command line, `server_address` and `server_port` is separate values.

Reply:
@ginnun OK, in that case, would it be better to remove this question: "What is the Blockfrost Platform `server-address` configuration parameter used for?" Maybe we don't need it in the FAQ.

Reply:
For now, I've removed the question: "What is the Blockfrost Platform server-address configuration parameter used for?"